### ISSUE TITLE
Switch loader class

### ISSUE THREAD

**Issue by spelkey-ucd** [UCDavisLibrary/forminator-addon-rt#9]
Getting the following notice:

```Forminator_Addon_Loader class is deprecated and will be removed soon. Use Forminator_Integration_Loader instead.```

**Comment by spelkey-ucd** [comment]
this has proven more difficult than anticipated. so, for the time being, i'm going to put a pin in it and hold off on updating forminator. hopefully they have some sort of upgrade guide.

here are some things i tried/might be important:
- update addon parent class to Forminator_Integration
- Update loader class to Forminator_Integration_Loader
- hooks and settings should have the following naming convention: `Forminator_' . ucfirst( $addon_slug ) . '_' . ucfirst( $module_type ) . '_Settings`
- Addonloader loader might need replacing in confirm_access method
- Add protected $_icon = ''; and protected $_image = ''; to constructor
- is_form_connected becomes is_module_connected - update signature
- replace Forminator_Addon_Form_Hooks_Abstract with Forminator_Integration_Form_Hooks
- replace Forminator_Addon_Form_Settings_Abstract with Forminator_Integration_Form_Settings

**Comment by spelkey-ucd** [comment]
I successfully updated code to work with new class API. Here is what I did:

## Addon Class

Change forminator classes
 - `Forminator_Addon_Abstract` to `Forminator_Integration`
 - `Forminator_Addon_Loader` to `Forminator_Integration_Loader`

Static instance updates:
- update `private static $_instance = null;` to `protected static $instance = null;`
- remove `get_instance` method
- remove `$_form_settings` property
- remove `$_form_hooks` property


Assets updates:
- override parent `addon_path` method - needs to be defined as string
- override parent `assets_path` method - needs to be defined as string
- remove `$this->_icon ` and `$this->_image` from constructor
- remove `_icon_x2`, `_full_path`, and `_image_x2` property declarations
- rename image in assets dir to `icon.png` and `image.png`
  
```php
  public function addon_path() : string  {
    return forminator_addon_rt_url();
  }

  public function assets_path() : string {
    return forminator_addon_rt_assets_url();
  }
```

Update to form connection
- remove `is_form_connected` entirely. parent class handles this.

## form settings class
class updates
- rename main class to format `'Forminator_' . ucfirst( $addon_slug ) . '_' . 'Form_Settings';`
- `Forminator_Addon_Form_Settings_Abstract` to `Forminator_Integration_Form_Settings`
- `Forminator_Addon_Abstract::get_button_markup` to `Forminator_Integration::get_button_markup`
- `Forminator_Addon_Abstract::get_template` to `Forminator_Integration::get_template`

method updates
- remove _construct
- `form_settings_wizards` becomes `module_settings_wizards`
- `get_form_settings_values` becomes `get_settings_values` 
- `save_form_settings_values` becomes `save_module_settings_values`

## Hooks class
class updates
- `Forminator_Addon_Form_Hooks_Abstract` becomes `Forminator_Integration_Form_Hooks`
- update constructor to `__construct(Forminator_Integration $addon, int $module_id)`, and change `form_id` to `module_id`
- update all references to `this->form_id` to `$this->module_id`
- rename main class name to format `'Forminator_' . ucfirst( $addon_slug ) . '_' . 'Form_Hooks';`

method updates
- `public function on_export_render_title_row(){ ` becomes `public function on_export_render_title_row() : array {`
- `on_form_submit` becomes `on_module_submit`
- `$this->form_settings_instance` becomes `$this->settings_instance`
- `$this->_submit_form_error_message ` becomes `$this->submit_error_message`
- `$this->custom_form` becomes `$this->module`

**Comment by spelkey-ucd** [comment]
https://github.com/UCDavisLibrary/forminator-addon-rt/commit/b55754cab84b2877ce788aee894fd5218e712f2c

**Comment by spelkey-ucd** [comment]
released v2.0.0
